TestSwitchApi/TestSwitchApi/Controllers/SessionsController.cs

```diff
@@ -31,7 +31,15 @@ public ActionResult<CandidateTestModel> AddTestSubmission([FromRoute] string tok
             var candidate = _candidates.GetCandidateByGuid(tokenId);
             if (candidate!=null)
             {
-                return _submissions.AddTestSubmission(candidate.Id, newSubmission);
+                try
+                {
+                    _submissions.AddTestSubmission(candidate.Id, newSubmission);
+                    return StatusCode(200);
+                }
+                catch
+                {
+                    return StatusCode(500);
+                }
```

Comment:
this try-catch doesn't seem to add anything.
By default, if there is an uncaught error then ASP will return a 500 response anyway.